compiler/modules/nback.llvm/llvm.bal

```diff
@@ -531,6 +540,19 @@ function createLine(string[] words, string indent = "") returns string {
     return string:concat(indent, ...parts);
 }
 
+function incrementName(string name, int count) returns string{
+    if count == 0 {
+       return name + "." + count.toString(); 
+    } else {
+        int? splitPoint = name.lastIndexOf(".");
```

Comment:
In this sort of case, just do `int splitPoint = <int>name.lastIndexOf(".");`

But I don't think you need this with the right data structure.

Reply:
Fixed by changing variable name store to map